src/pages/Preferences.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useParams, Link } from "react-router-dom";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { SideNav, SideNavGroup, SideNavItem } from "@leafygreen-ui/side-nav";
+import get from "lodash/get";
+import { paths } from "constants/routes";
+import { PageWrapper } from "components/styles";
+import { PreferencesTabs } from "pages/preferences/PreferencesTabs";
+import { GET_USER_SETTINGS } from "gql/queries/get-user-settings";
+import {
+  GetUserSettingsQuery,
+  GetUserSettingsQueryVariables,
+} from "gql/generated/types";
+
+export const Preferences: React.FC = () => {
+  const { tab } = useParams<{ tab: string }>();
+  const { data, loading, error } = useQuery<
+    GetUserSettingsQuery,
+    GetUserSettingsQueryVariables
+  >(GET_USER_SETTINGS);
+
+  const userSettings = get(data, "data.userSettings");
+  return (
+    <PageWrapper>
+      <PageContainer>
+        <SideNav>
+          <SideNavGroup header="Preferences">
+            <PaddedSideNavItem
+              active={tab === "profile"}
+              to={`${paths.preferences}/profile`}
```

Comment:
we should save the profile, notifications and cli in an enum so we can keep track of and update them easier